firefox_accounts/firefox_accounts.model.lkml

```diff
@@ -1,7 +1,7 @@
 connection: "telemetry"
 label: "Firefox Accounts"
 include: "//looker-hub/firefox_accounts/explores/*"
-include: "//looker-hub/firefox_accounts/dashboards/*"
-include: "views/*"
-include: "explores/*"
-include: "dashboards/*"
\ No newline at end of file
+# include: "//looker-hub/firefox_accounts/dashboards/*"
+# include: "views/*"
+# include: "explores/*"
+# include: "dashboards/*"
```

Comment:
@fbertsch these are the lines I commented out - tagging you for context around the errors I encountered earlier.

Reply:
Great, sounds good. I made similar changes in https://github.com/mozilla/looker-spoke-default/pull/102 which you may have to rebase on top of.